modules/statics-deploy/main.tf

```
@@ -157,6 +160,38 @@ data "aws_iam_policy_document" "access_sqs_queue" {
   }
 }
 
+#
+# Lambda permissions to create deployment
+#
+data "aws_iam_policy_document" "create_deployment" {
+  statement {
+    actions = [
+      "apigateway:POST",
+      "ec2:DescribeSecurityGroups",
+      "ec2:DescribeSubnets",
+      "ec2:DescribeVpcs",
+      "iam:AttachRolePolicy",
+      "iam:CreateRole",
+      "iam:PassRole",
+      "lambda:AddPermission",
+      "lambda:CreateFunction",
+      "logs:CreateLogGroup",
+      "logs:PutRetentionPolicy"
+    ]
+    resources = ["*"]
+  }
+
+  statement {
+    actions   = ["s3:PutObject"]
+    resources = ["${var.proxy_config_bucket_arn}/*"]
+  }
+
+  statement {
+    actions   = ["dynamodb:PutItem"]
+    resources = [var.proxy_config_table_arn]
```

This line cause "Error: Null value found in list".

```
│ Error: Null value found in list
│ 
│   with module.tf_next.module.statics_deploy.data.aws_iam_policy_document.create_deployment,
│   on ../../modules/statics-deploy/main.tf line 192, in data "aws_iam_policy_document" "create_deployment":
│  192:     resources = [var.proxy_config_table_arn]
│ 
│ Null values are not allowed for this attribute value.
```

When the value is null, empty string should be set, 

```suggestion
    resources = [var.proxy_config_table_arn != null ? var.proxy_config_table_arn : ""]
```

or use `multiple_deployments` variable like #185 .

---

Fixed.